File: backend/src/utils/blood_topics.ts
const bloodGroups = [
  'A+',
  'A-',
  'B+',
  'B-',
  'O+',
  'O-',
  'AB+',
  'AB-',
];

export const bloodGroupToTopic = (bg: string) => bg.replaceAll('+', 'Plus').replaceAll('-', 'Minus');

File: app/lib/utils/blood_topics.dart
const List<String> bloodGroups = [
  'A+',
  'A-',
  'B+',
  'B-',
  'AB+',
  'AB-',
  'O+',
  'O-',
];

String bloodGroupToTopic(String bg) => bg.replaceAll('+', 'Plus').replaceAll('-', 'Minus');
